recipes-nemomobile/mce/mce/10mce.ini

```diff
@@ -16,7 +16,7 @@ ModulePath=/usr/lib/mce/modules
 # to avoid unnecessary brightness fluctuations on mce startup
 #
 # Note: the name should not include the "lib"-prefix
-Modules=radiostates;display;keypad;led;battery-statefs;inactivity;alarm;callstate;audiorouting;powersavemode;cpu-keepalive;doubletap;packagekit;sensor-gestures;bluetooth;memnotify;usbmode
+Modules=radiostates;display;keypad;led;battery-udev;inactivity;alarm;callstate;audiorouting;powersavemode;cpu-keepalive;doubletap;packagekit;sensor-gestures;bluetooth;memnotify;usbmode
```

Comment:
I guess I'm lacking context here. I thought you were replacing statefs on all watches, why is this patch only for mtk6580 ?

Reply:
mce.ini is by default provided by `meta-asteroid` as in, it's copied from the repository.
Apparently, the mtk6580 hybris layer uses a custom config file. As such it needs to be altered :wink: 

Relevant upstream commit: https://git.sailfishos.org/mer-core/mce/commit/d2e2e0aa74aecce32c98d80f457dcff5234e880e

Reply:
oh I see, I hope I wrote a more descriptive commit desc in https://github.com/AsteroidOS/meta-mtk6580-hybris/commit/2ed980214aa981ee7d4ddae75d03d12efb4ce4a9
LGTM then